mock-sdk/src/test/java/io/mosip/mock/sdk/impl/SampleSDKTest.java

```diff
@@ -122,7 +122,7 @@ private BiometricRecord xmlFileToBiometricRecord(String path) throws ParserConfi
         DocumentBuilder dBuilder = dbFactory.newDocumentBuilder();
         Document doc = dBuilder.parse(fXmlFile);
         doc.getDocumentElement().normalize();
-        LOGGER.debug("Root element :" + doc.getDocumentElement().getNodeName());
+        LOGGER.debug("Root element :",  doc.getDocumentElement().getNodeName());
         Node rootBIRElement = doc.getDocumentElement();
         NodeList childNodes = rootBIRElement.getChildNodes();
         for (int temp = 0; temp < childNodes.getLength(); temp++) {
```

Comment:
Do we need unit test case changes for this ?

Reply:
Since we are not sending original ISO in the testcase, we can not test it for the flags sdk_check_iso_timestamp_format